Content.Shared/CCVar/CCVars.cs

```diff
@@ -286,6 +286,19 @@ public sealed class CCVars : CVars
         public static readonly CVarDef<int> NukeopsPlayersPerOp =
             CVarDef.Create("nukeops.players_per_op", 5);
 
+        /*
+         * Zombie
+         */
+
+        public static readonly CVarDef<int> ZombieMinPlayers =
+            CVarDef.Create("zombie.min_players", 25);
+
+        public static readonly CVarDef<int> ZombieMaxInfected =
+            CVarDef.Create("zombie.max_infected", 4);
+
+        public static readonly CVarDef<int> ZombiePlayersPerInfected =
+            CVarDef.Create("zombie.players_per_infected", 7);
```

Comment:
isnt it basically always gonna be 4 then, since the min is 25 and it would max out at 28